trio/tests.py

```diff
@@ -0,0 +1,36 @@
+import importlib
+import sys
+from typing import Any
+
+# don't ask which of the _tests imports is necessary
+import trio._tests
+
+from . import _tests
```

Comment:
I'm a bit confused by this, won't the namespaces shadow and so the first is useless?